backend/internal/controller/adapter/shot_message_test.go

```diff
@@ -134,3 +139,61 @@ func TestDecodeTeam(t *testing.T) {
 		})
 	}
 }
+
+func Test_checkIfShotAtGoal(t *testing.T) {
+	type args struct {
+		areaID int
+		teamID int
+	}
+	tests := []struct {
+		name           string
+		args           args
+		expectedResult bool
+	}{
+		{
+			name: "should return true when white team shots at blue team goal",
+			args: args{
+				areaID: config.WhiteTeamGoalArea,
+				teamID: config.TeamBlue,
+			},
+			expectedResult: true,
+		},
+		{
+			name: "should return true when blue team shots at white team goal",
+			args: args{
+				areaID: config.BlueTeamGoalArea,
+				teamID: config.TeamWhite,
+			},
+			expectedResult: true,
+		},
+		{
+			name: "should return false when blue team shots at their own goal",
+			args: args{
+				areaID: config.WhiteTeamGoalArea,
+				teamID: config.TeamWhite,
+			},
+			expectedResult: false,
+		},
```

Comment:
You type that "white team shots at blue area" and in args there is ID of the blue team and area of white goal. I think it may be a typo.

Reply:
Yes, this is a small mistake

Reply:
Described issue has been fixed.